contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -1051,6 +1051,25 @@ void USDScene::writeAttribute( const SceneInterface::Name &name, const Object *a
 				attribute.Set( DataAlgo::toUSD( data ), time );
 			}
 		}
+		else if( const IECore::CompoundObject *globals = runTimeCast<const CompoundObject>( attribute ) )
+		{
+			if( name.string() == "gaffer:globals" )
```

Comment:
A non-nested `if( name.string() == "gaffer:globals" )` at the top level would be clearer than nesting this inside the generic attribute handling (and it'd make more sense to test for the CompoundObject after determining what we're expecting).